src/Database/Adapter/MariaDB.php

```diff
@@ -1311,7 +1312,7 @@ public function getAttributeWidth(Document $collection): int
                     $total += 19; // 2022-06-26 14:46:24
                     break;
                 default:
-                    throw new Exception('Unknown Type');
+                    throw new DatabaseException('Unknown Type');
```

Comment:
```suggestion
                    throw new DatabaseException('Unknown Type. Must be one of ${Database::VAR_BOOLEAN}, ...');
```

Can we format this error message to be more descriptive. Let's add all the types in the message